models/Category.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
```

Comment:
You can do:
`const { Schema } = mongoose`

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Learn more about destructuring

Reply:
Okay